packages/cli/package.json

```diff
@@ -51,11 +51,11 @@
     "yaml": "^2.1.1"
   },
   "devDependencies": {
-    "@types/jest": "29.5.3",
+    "@types/jest": "29.5.5",
     "@types/node-fetch": "2.6.4",
     "@types/pouchdb": "^6.4.0",
     "renamer": "^4.0.0",
-    "ts-node": "^10.9.1",
+    "ts-node": "10.8.1",
```

Comment:
This is the one dependency I've seen so far that's going down, not up. Is there something in `10.9.x` that's incompatible with the account portal?